kafka/init.lua

```diff
@@ -359,6 +386,36 @@ function Producer:dump_conf()
     return self._producer:dump_conf()
 end
 
+function Producer:metadata(options)
+    if self._producer == nil then
+        return
+    end
+
+    local timeout_ms = DEFAULT_TIMEOUT_MS
+    if options ~= nil and options.timeout_ms ~= nil then
+        timeout_ms = options.timeout_ms
+    end
+
+    local topic
+    if options ~= nil and options.topic ~= nil then
```

Comment:
Is it `box.NULL` protection? Why it's impossible to use options.topic as is?

Reply:
Consider it as box.NULL protection. Just to be sure.